doc/mkdocs/docs/api/basic_json/default_object_comparator_t.md

```diff
@@ -0,0 +1,18 @@
+# <small>nlohmann::basic_json::</small>default_object_comparator_t
+
+```cpp
+using default_object_comparator_t = std::less<StringType>;  // until C++14
+
+using default_object_comparator_t = std::less<>;            // since C++14
+```
+
+The default comparator used by [`object_t`](object_t.md).
+
+Since C++14 a transparent comparator is used which prevents unnecessary string construction.
```

Comment:
Maybe add half a sentence where it is used?

Reply:
"Since C++14 a transparent comparator is used which prevents unnecessary string construction when looking up a key in an object."